pages/event/[eventSlug]/schedules/index.vue

```
@@ -37,15 +39,19 @@ definePageMeta({
 });
 
 const route = useRoute();
-const eventStore = useEventStore();
-const userStore = useUserStore();
 const eventSlug = route.params.eventSlug as string;
+const userStore = useUserStore();
+
+const questionnaireId = route.query.questionnaireId as string | undefined;
+const status = route.query.status as "able-to-donate" | "unable-to-donate" | undefined;
```

ESLint

[error] Replace `·"able-to-donate"·|·"unable-to-donate"` with `⏎··|·"able-to-donate"⏎··|·"unable-to-donate"⏎·`

(prettier/prettier)